Description:
  Recent versions of sox doesn't support the -w option. Remove it from the
  command line. 
Author: Michael Kaaden <kaaden@galileo.franken.de>
Bug-Debian: http://bugs.debian.org/519062
Last-Update: 2010-07-07

--- a/scripts/cs_helpers.pyin
+++ b/scripts/cs_helpers.pyin
@@ -229,7 +229,7 @@
 		elif (mail_type=="la"): # voice file
 			# la -> wav
 			# don't use stdout as sox needs a file to be able to seek in it otherwise the header will be incomplete
-			ret = os.spawnlp(os.P_WAIT,"sox","sox",attachment,"-w",basename+"wav")
+			ret = os.spawnlp(os.P_WAIT,"sox","sox",attachment,basename+"wav")
 			if (ret or not os.access(basename+"wav",os.R_OK)):
 				raise "conv-error","Error while calling sox. Not installed?"
 			filepart = email.MIMEAudio.MIMEAudio(open(basename+"wav").read(),"x-wav",email.Encoders.encode_base64,name=os.path.basename(basename)+"wav")
